src/archivematicaCommon/lib/databaseFunctions.py

```diff
@@ -106,53 +203,72 @@ def logTaskCreatedSQL(taskManager, commandReplacementDic, taskUUID, arguments):
     taskexec = taskManager.execute
     fileName = os.path.basename(os.path.abspath(commandReplacementDic["%relativeLocation%"]))
 
-    databaseInterface.runSQL("""INSERT INTO Tasks (taskUUID, jobUUID, fileUUID, fileName, exec, arguments, createdTime)
-        VALUES (%s, %s, %s, %s, %s, %s, %s)""",
-        (taskUUID, jobUUID, fileUUID, fileName,
-         taskexec, arguments, databaseInterface.getUTCDate())
-    )
-
-def logTaskAssignedSQL(taskUUID, client, date):
-    databaseInterface.runSQL(
-        """UPDATE Tasks SET startTime=%s, client=%s WHERE taskUUID=%s;""",
-        (date, client, taskUUID)
-    )
+    Task.objects.create(taskuuid=taskUUID,
+                        job_id=jobUUID,
+                        fileuuid=fileUUID,
+                        filename=fileName,
+                        execution=taskexec,
+                        arguments=arguments,
+                        createdtime=getUTCDate())
 
 def logTaskCompletedSQL(task):
+    """
+    Fetches execution data from the completed task and logs it to the database.
+    Updates the entry in the Tasks table with data in the provided task.
+    Saves the following fields: exitCode, stdOut, stdError
+
+    :param task:
+    """
     print "Logging task output to db", task.UUID
     taskUUID = task.UUID.__str__()
     exitCode = task.results["exitCode"].__str__()
```

Comment:
I have a feeling it was always being coerced into a string and then back into an integer in the past; could adjust it in a followup PR to not do that?